### PR TITLE
Return index in simpleFragments, add dummy preview

### PR DESCRIPTION
**Pull request by aliciakw**
Return index value in SimpleFragment output so its easy to render the collection in a stable order.

Also added a preview of the output to the Dummy page, just for fun.

### CODE REVIEW

File: app/services/extension.js [hunk 1]
```diff
@@ -66,13 +67,27 @@ export default Service.extend({
     set(this, 'extension', extension);
     this.loadSchemaFromShorthand();
     this.syncFragmentsToSchema();
+    this.persist();
```
Comment (hhff) [inline app/services/extension.js]: Why call persist here?

File: dist/index.html [hunk 2]
```diff
@@ -47,8 +48,8 @@
       window.contentfulExtension.init(window.contentfulFragment.init);
     </script>
 
-    <script src="https://rawgit.com/sanctuarycomputer/contentful-fragment/master/dist/assets/vendor-41efc319637f06f350e3043c1913afe6.js"></script>
-    <script src="https://rawgit.com/sanctuarycomputer/contentful-fragment/master/dist/assets/contentful-fragment-42ef4b44ac540fbd1a92dcd273a68b42.js"></script>
```
Comment (hhff) [inline dist/index.html:51]: can you run `ember build --production` and commit the result?

File: app/templates/index.hbs [hunk 3]
```diff
@@ -178,3 +178,10 @@
     Setup Fragment
   {{/link-to}}
 {{/if}}
+
+{{#if extension.data._preview }}
+<div>
+  <p>OUTPUT:</p>
+  <pre class="json-preview">{{extension.data._preview}}</pre>
+</div>
+{{/if}}
```
Comment (hhff) [inline app/templates/index.hbs]: Cool!

File: app/services/extension.js [hunk 4]
```diff
@@ -66,13 +67,27 @@ export default Service.extend({
...
   },
 
   setSetting(key, value) {
     set(this, 'data._settings', get(this, 'data._settings') || {});
     set(this, `data._settings.${key}`, value);
   },
 
+  renderPreview(simpleFragments) {
+    if (this.extension._isDummy) {
```
Comment (hhff) [inline app/services/extension.js]: why are you checking `_isDummy` here? This means it will only work on local development

Comment (hhff) [reply app/services/extension.js]: or is this a dev tool?

**Review by hhff — CHANGES_REQUESTED** [review]

File: app/services/extension.js (hunk 4 above)
Comment (aliciakw) [reply app/services/extension.js]: I was being conservative & leaving it as a dev tool. 
But could totally open it up to production if you think its a good idea!

File: app/services/extension.js (hunk 1 above)
Comment (aliciakw) [reply app/services/extension.js]: simple fragments weren't being generated on the initial render... maybe I should just call `this.makeSimpleFragments();` directly though

File: app/services/extension.js (hunk 4 above)
Comment (aliciakw) [reply app/services/extension.js]: I guess if its in prod, maybe should be collapsible or something

**Comment by aliciakw** [comment]
updated @hhff! pls let me know what you think

File: app/services/extension.js (hunk 4 above)
Comment (hhff) [reply app/services/extension.js]: mmm yeah lets leave as dev, a lil confusing for non devs

**Comment by hhff** [comment]
looks like you still need to run `ember build --production`

File: app/services/extension.js (hunk 4 above)
Comment (hhff) [reply app/services/extension.js]: I feel like this preview should render the entire payload, rather than just simple fragemnts

Comment (hhff) [reply app/services/extension.js]: This also shouldn't attach to `data`, as `data` is the key that actually serializes to Contentful's column

**Review by hhff — CHANGES_REQUESTED** [review]

**Comment by aliciakw** [comment]
Updated so that the full payload is being previewed. Also now its collapsible.

screen shot 2018-10-09 at 6 12 14 pmscreen shot 2018-10-09 at 6 09 35 pm

File: app/styles/app.css [hunk 5]
```diff
@@ -96,6 +96,11 @@ hr {
   letter-spacing: 0.1em;
 }
 
+.cf-card-field {
+  text-overflow: ellipsis;
+  max-width: 20rem;
+}
+
```
Comment (aliciakw) [inline app/styles/app.css:103]: this fixes a bug where, if you have a really long string (like a long url) the edit button disappears

**Comment by aliciakw** [comment]
Adding husky in https://github.com/sanctuarycomputer/contentful-fragment/pull/10, & rebased off of that. Mind taking another look @hhff ?

**Pull request merged**